packages/deployment/app/cloudformation.template.yml

```diff
@@ -160,7 +179,11 @@ Resources:
       Action: lambda:InvokeFunction
       Principal: sns.amazonaws.com
       SourceArn:
+        {{#if ../this.arn}}
+        {{../this.arn}}
```

Comment:
Did you test this? The yaml might complain about the line break here.

Reply:
I did test